cypress/integration/common/stepDefinations/UI/IDIR_No_GOLD_onlyTeams_StepDef.js

```diff
@@ -0,0 +1,49 @@
+var a = 0;
```

Comment:
`a` and `b`? variable names can be much more meaningful.

Reply:
I've used it as a Flag and further used as a counter.